Modules/Core/Common/CMakeLists.txt

```diff
@@ -131,6 +131,7 @@ set(ITK_USE_TBB ${Module_ITKTBB})
 configure_file(src/itkConfigure.h.in itkConfigure.h)
 configure_file(src/itkConfigurePrivate.h.in itkConfigurePrivate.h)
 
```

Comment:
`if(ITK_WRAP_PYTHON)`

Reply:
Latest force-push adds this condition. I did not put it there initially as I thought that variable would be empty when wrapping is off. But I now realize that it comes from finding Python, and it is usually found even when wrapping is off.